cypress/support/step-definitions/search-steps.js

```diff
@@ -1,12 +1,25 @@
 import {
-  searchComponent, searchInput, searchInnerIcon, searchIcon, searchInputNoiFrame,
+  searchInput, searchDefault, searchIcon, searchInputNoiFrame,
+  searchDefaultInput, searchCrossIcon, searchButton, searchWitchButtonInput, searchDefaultInnerIcon,
 } from '../../locators/search';
-import { positionOfElement } from '../helper';
+import { BUTTON_DATA_COMPONENT_PREVIEW } from '../../locators/button/locators';
```

Comment:
what for we imported here this locator ?

Reply:
deleted